data-science-onramp/feature-engineering/feature_engineering.ipynb

```diff
@@ -1,4795 +1,1019 @@
 {
```

Comment:
<p>First sentence:</p><p> </p><p>Original:</p><p>The purpose of the holiday feature column is to represent a binary value of whether there is a holiday on a specific day or not</p><p> </p><p>Proposed:</p><p>The purpose of the holiday feature column is to represent a binary value <strong>for</strong> whether there is a holiday on a specific day or not, rather than the type of holiday</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/leahecole/python-docs-samples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>Done</p>